changelog.md

```diff
@@ -12,6 +12,7 @@ Saves from 5.x are not compatible with 6.0.
 * **[Flight Planning]** Added the ability to plan tankers for recovery on package flights.  AI does not plan.
 * **[Flight Planning]** Air to Ground flights now have ECM enabled on lock at the join point, and SEAD/DEAD also have ECM enabled on detection and lock at ingress.
 * **[Flight Planning]** AWACS flightplan changed from orbit to a racetrack to reduce data link disconnects which were caused by blind spots as a result of the bank angle. 
+* **[Flight Planning]** The Tornado GR4 will no longer be tasked for SEAD missions due to no access to ARMs.
```

Comment:
It's more a fix than a feature :) you can also feel free to squash the two commits to one. no need for a separate changelog commit.

Reply:
Thanks for noticing that! It's now in the correct place.